app/main.ipynb

```diff
@@ -216,15 +269,28 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
-   "source": [
-    "# write your code here"
-   ],
+   "source": "A.std(axis=1)",
```

Comment:
The standard deviation calculation is performed along the wrong axis. Use `axis=0` to get the standard deviation by the `y` axis.